test/unit/ngsiv2/general/startup-test.js

```diff
@@ -0,0 +1,147 @@
+                        name: 'pressure',
+                        type: 'Hgmm'
+                    }
+                ]
+            }
+        },
+        providerUrl: 'http://smartGondor.com',
+        deviceRegistrationDuration: 'P1M',
+        throttling: 'PT5S'
+    },
+    iotamMock;
+
+describe('Startup tests', function() {
+
+    describe('When the IoT Agent is started with environment variables', function() {
+        beforeEach(function() {
+            process.env.IOTA_CB_HOST = 'cbhost';
+            process.env.IOTA_CB_PORT = '1111';
+            process.env.IOTA_CB_NGSI_VERSION = 'v2';
+            process.env.IOTA_NORTH_HOST = 'localhost';
+            process.env.IOTA_NORTH_PORT = '2222';
+            process.env.IOTA_PROVIDER_URL = 'provider:3333';
+            process.env.IOTA_REGISTRY_TYPE = 'mongo';
+            process.env.IOTA_LOG_LEVEL = 'FATAL';
+            process.env.IOTA_TIMESTAMP = true;
+            process.env.IOTA_IOTAM_HOST = 'iotamhost';
+            process.env.IOTA_IOTAM_PORT = '4444';
+            process.env.IOTA_IOTAM_PATH = '/iotampath';
+            process.env.IOTA_IOTAM_PROTOCOL = 'PDI_PROTOCOL';
+            process.env.IOTA_IOTAM_DESCRIPTION = 'The IoTAM Protocol';
+            process.env.IOTA_MONGO_HOST = 'mongohost';
+            process.env.IOTA_MONGO_PORT = '5555';
+            process.env.IOTA_MONGO_DB = 'themongodb';
+            process.env.IOTA_MONGO_REPLICASET = 'customReplica';
+            process.env.IOTA_DEFAULT_RESOURCE = '/iot/custom';
+
+            nock.cleanAll();
+
+            iotamMock = nock('http://iotamhost:4444')
+                .post('/iotampath')
+                .reply(200,
+                    utils.readExampleFile('./test/unit/ngsiv2/examples/iotamResponses/registrationSuccess.json'));
+        });
+
+        afterEach(function() {
+            delete process.env.IOTA_CB_HOST;
+            delete process.env.IOTA_CB_PORT;
+            delete process.env.IOTA_CB_NGSI_VERSION;
+            delete process.env.IOTA_NORTH_HOST;
+            delete process.env.IOTA_NORTH_PORT;
+            delete process.env.IOTA_PROVIDER_URL;
+            delete process.env.IOTA_REGISTRY_TYPE;
+            delete process.env.IOTA_LOG_LEVEL;
+            delete process.env.IOTA_TIMESTAMP;
+            delete process.env.IOTA_IOTAM_HOST;
+            delete process.env.IOTA_IOTAM_PORT;
+            delete process.env.IOTA_IOTAM_PATH;
+            delete process.env.IOTA_IOTAM_PROTOCOL;
+            delete process.env.IOTA_IOTAM_DESCRIPTION;
+            delete process.env.IOTA_MONGO_HOST;
+            delete process.env.IOTA_MONGO_PORT;
+            delete process.env.IOTA_MONGO_DB;
+            delete process.env.IOTA_MONGO_REPLICASET;
+            delete process.env.IOTA_DEFAULT_RESOURCE;
+        });
+
+        afterEach(function(done) {
+            iotAgentLib.deactivate(done);
+        });
+
+        it('should not start and raise a MISSING_CONFIG_PARAMS error', function(done) {
```

Comment:
To which missing parameter this test refers?

Reply:
Fixed in bccb280945f044c24c35516a3c3617059f0c6991